### ISSUE TITLE
Azure AI Search - vector search with text-embedding-3-large not working

### ISSUE THREAD

**Issue by jucastag** [Azure/gpt-rag#110]
I created an index manually ingesting json's from blob storage. Json are simple text with 'contentVector' field with embeddings. I tried diferent dimensions for testing: 256, 1536, 2048 and 3072.  While configuring the index 'contentVector' field It only allows me to use 2048 as maximum dimension and not 3072. Also.. while configuring the vectorizer it doesn't allow me to set my 'text-embedding-3-large' deployed model. Only text-embedding-ada-002' appears as an option. In terms.. the vector search through azure portal doesnt work with error InvalidVectorQuery: Query vector length must match the dimensionality of the field 'contentVector' in the vector field list"

**Comment by framigni** [comment]
Hi @jucastag related to your problem, we have tried to use a different Index as well, but unable to make the Orchestrator working with the new Index (see Issue #111 ) . Have you had a chance to test the RAG with your new Index ?

**Comment by jucastag** [comment]
@framigni yes..  we've tested the app with different indexes in the past. But we tested this by creating different indexes using new runs of setup.py of the ingestion code. Make sure the new created index has the same schema as the orchestrator is expecting to find.. specially the retireval functionality. Use log stream of the orchestrator function in azure portal to debugg. We are able to create vecttorizers... Are you sure you have the latest infra deployed? remember to set contentVector type of data as edm collection single. Also the new infra uses different search and openai api version. that could also generate issues.

**Comment by redvivi** [comment]
Any update on this?
